cli/src/pcluster/validators/dev_settings_validators.py

```diff
@@ -0,0 +1,62 @@
+# Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance
+# with the License. A copy of the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "LICENSE.txt" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES
+# OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions and
+# limitations under the License.
+import json
+
+from pcluster.validators.common import FailureLevel, Validator
+from pcluster.validators.utils import dig, is_boolean_string, str_to_bool
+
+EXTRA_CHEF_ATTRIBUTES_PATH = "DevSettings/Cookbook/ExtraChefAttributes"
```

Comment:
I think we should also specify the Phase of ParallelCluster this will affect in the Description of the PR as `ExtraChefAttributes` are used in both Cluster CRUD and Build Image (even though its clear to us in the team)